http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1382,10 +1656,9 @@ components:
           schema:
             $ref: "#/components/schemas/CreateRuntimeRequest"
           example:
-            runtimeConfig:
-              cloudService: gce
-              machineType: n1-standard-4
-              diskSize: 100
+            size: 500
```

Comment:
is the size increase intentional?

Reply:
Hm.. let's keep `CreateRuntimeRequest` unchanged in this PR, I am updating it in my PR